pkg/metrics/ensuarance.go

```diff
@@ -258,3 +301,23 @@ func PodResourceUpdateErrorCounterInc(subComponent SubComponent, stepName StepLa
 func ExecutorEvictCountsInc() {
 	executorEvictCounts.Inc()
 }
+
+func UpdateNodeCpuCannotBeReclaimedSeconds(node string, value float64) {
```

Comment:
Is the node info in the prometheus metric already? "node string" can  be removed

Reply:
It should be so, but our internal prometheus didn't add node info, so I added node info to the crane-agent by default